.github/actions/on_device_tests/action.yaml

```diff
@@ -132,6 +132,10 @@ runs:
               python3 ${GITHUB_WORKSPACE}/.github/scripts/generate_crash_report.py "${log_path}" "${xml_path}"
               exit 1
             fi
+          elif grep -e '<failure' -e '<error' ${xml_files}; then
```

Comment:
Is it possible for these tags to exist in the output xml if all tests succeed? Hopefully not?